src/effects/FindClipping.cpp

```diff
@@ -96,8 +96,8 @@ bool EffectFindClipping::GetAutomationParameters(CommandParameters & parms) cons
 
 bool EffectFindClipping::SetAutomationParameters(const CommandParameters & parms)
 {
-   ReadAndVerifyInt(Start);
-   ReadAndVerifyInt(Stop);
+   ReadParam(Start);
```

Comment:
Now that `ReadParam` resolves types at compile time, we have no more need for the many `ReadAndVerifyInt`, `ReadAndVerifyBool `etc.  - however, in the process we lost the verb "verify" in the function name , which is what ReadParam actually does. So, could we rename `ReadParam` to `ReadAndVerify`?

Reply:
The commit "Remove unused macros and the cache in EffectParameter" ultimately eliminates these macros.  This is just one simplifying step in my transformation.

Reply:
ok